### PR TITLE
(DED-591) Create a new navbar in docs

### PR DESCRIPTION
**Pull request by arturfracala**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira](https://livechatinc.atlassian.net/browse/DED-591)

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/components/core/Navbar/Platform.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <Wrapper height="520px">
+        <div>
+          {platform.leftColumn.map((component) => (
+            <DropdownItem
+              title={component.title}
+              description={component.description}
+              to={component.link}
+              image={component.image}
+            />
+          ))}
```
Comment (AdalbertusLiver) [inline src/components/core/Navbar/Platform.js:39]: missing `key` prop when mapping

File: src/components/core/Navbar/Platform.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        <div>
+          {platform.rightColumn.map((component) => (
+            <DropdownItem
+              title={component.title}
+              description={component.description}
+              to={component.link}
+              image={component.image}
+              badge={component.badge}
+            />
+          ))}
```
Comment (AdalbertusLiver) [inline src/components/core/Navbar/Platform.js:51]: missing `key` prop when mapping

File: src/components/core/Navbar/Resources.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    <Dropdown label="Resources">
+      <Wrapper height="520px">
+        <div>
+          {resources.map((resource) => (
+            <DropdownItem
+              title={resource.title}
+              description={resource.description}
+              to={resource.link}
+            />
+          ))}
```
Comment (AdalbertusLiver) [inline src/components/core/Navbar/Resources.js:39]: missing `key` prop when mapping

File: src/components/core/Navbar/Resources.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+        <Separator />
+        <div>
+          <TitleWrapper>CONNECT</TitleWrapper>
+          {connects.map((connect) => (
+            <Connect
+              title={connect.title}
+              image={connect.image}
+              link={connect.link}
+            />
+          ))}
```
Comment (AdalbertusLiver) [inline src/components/core/Navbar/Resources.js:50]: missing `key` prop when mapping

**Review by AdalbertusLiver — CHANGES_REQUESTED** [review]

**Review by SamborGornicz — APPROVED** [review]

**Review by AdalbertusLiver — APPROVED** [review]

**Pull request merged**